index.html

```diff
@@ -90,7 +91,47 @@ <h2>Artwork Design</h2>
         <!-- CONTACT -->
 
             <section class="contact">
+                <div class="formcontainer"> 
+                    <div class="formimage">
+                        <img src="images/form-img.png">
+                    </div>
+                    
+                    <div class="form">
+                        <h2> Interested to work together? Let's talk</h2>
+                        <form>
+                            <div class="formelement"><label for="name">
```

Comment:
It is suggested to use a camel case for the classes in HTML. (e.g. "formElement")

Reply:
Also, it seems this class ("formelement") is not used further in the CSS file. Could you please review if it is needed?

Reply:
Not needed so we removed it.